### DISCUSSION TITLE
Duplicate item on [enter]

### DISCUSSION THREAD

**Discussion by MuteKnowLESS** [johannes-wolf/rpnspire#31] [Ideas]
Hello, I was wondering if you could implement a feature where if you press enter and there are no input characters,  than it duplicates the last item in the stack.
something like this:

```
#in python because I dont know lua
if len(user_input) == 0 and enter == True:
    stack.append(stack[-1])
```

**Comment by johannes-wolf** [comment]
Implemented this with https://github.com/johannes-wolf/rpnspire/commit/1ac8428c4b13c675990b847f5b279f1dde99fb6a :)
See PR: https://github.com/johannes-wolf/rpnspire/pull/32
